src/dataclients/UserClient.js

```diff
@@ -0,0 +1,18 @@
+import { API_URL, handleErrors } from 'utils/FetchUtils';
+
+/**
+ * API to create a new field
+ * @param {object} payload
+ */
+export const createUser = async (payload) => {
```

Comment:
Can you please take up deletion of user as well as the next change. That also falls under bare minimum I think :)